### PR TITLE
Use the css style instead of title_fg_color and title_bg_color.

### PR DESCRIPTION
**Pull request by bachandi**

### CODE REVIEW

File: lib/traffacct.pm [hunk 1]
```diff
@@ -686,10 +684,8 @@ sub traffacct_cgi {
 		if(!$silent) {
 			print("  <table cellspacing='5' cellpadding='0' width='1' bgcolor='$colors->{graph_bg_color}' border='1'>\n");
 			print("  <tr>\n");
-			print("  <td bgcolor='$colors->{title_bg_color}' colspan='1'>\n");
-			print("  <font face='Verdana, sans-serif' color='$colors->{title_fg_color}'>\n");
-			print("    <b>&nbsp;&nbsp;Network traffic<b>\n");
-			print("  </font>\n");
+			print("  <td class='td-title' colspan='1'>\n");
```
Comment (mikaku) [inline lib/traffacct.pm:687]: You removed accidentally the variable `$traffacct->{graphs_per_row}`.

Comment (bachandi) [reply lib/traffacct.pm:687]: It was `colspan='1'` in this line before or am I missing something?

Comment (mikaku) [reply lib/traffacct.pm:687]: Oh, yes. I misread it. Sorry.

**Comment by mikaku** [comment]
I saw you also fixed some closing tags.
Thank you very much.

**Comment by mikaku** [comment]
Hi,

After this PR was applied I'm seeing a default color with the footer links in all graphs:
BIND

They should look like this:
bind

Probably the colors definition for links is missing in the class `td-title`.
Can you, please, take it a look?

**Pull request merged**